subsys/bluetooth/audio/has.c

```diff
@@ -70,33 +71,42 @@ static ssize_t read_features(struct bt_conn *conn, const struct bt_gatt_attr *at
 }
 
 /* Hearing Access Service GATT Attributes */
-BT_GATT_SERVICE_DEFINE(has_svc,
-	BT_GATT_PRIMARY_SERVICE(BT_UUID_HAS),
-	BT_AUDIO_CHRC(BT_UUID_HAS_HEARING_AID_FEATURES,
-		      BT_GATT_CHRC_READ,
-		      BT_GATT_PERM_READ_ENCRYPT,
-		      read_features, NULL, NULL),
-#if defined(CONFIG_BT_HAS_PRESET_SUPPORT)
-	BT_AUDIO_CHRC(BT_UUID_HAS_PRESET_CONTROL_POINT,
 #if defined(CONFIG_BT_EATT)
-		      BT_GATT_CHRC_WRITE | BT_GATT_CHRC_INDICATE | BT_GATT_CHRC_NOTIFY,
+#define CONTROL_POINT_METHODS (BT_GATT_CHRC_WRITE | BT_GATT_CHRC_INDICATE | BT_GATT_CHRC_NOTIFY)
```

Comment:
```suggestion
#define CONTROL_POINT_PROP (BT_GATT_CHRC_WRITE | BT_GATT_CHRC_INDICATE | BT_GATT_CHRC_NOTIFY)
```

It's GATT Characteristic Properties bitfield as defined in Core